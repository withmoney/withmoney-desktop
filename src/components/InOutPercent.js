import React from 'react';
import PercentBar from './PercentBar';

const InOutPercent = () => (
  <div className="inout-percent">
    <div className="inout-percent__type">
      <div className="inout-percent__info">
        <div className="inout-percent__total">Total In: $ 5,000.00</div>
        <div className="inout-percent__done">$450.34</div>
      </div>
      <PercentBar type="in" percent="33%" />
    </div>
    <div className="inout-percent__type">
      <div className="inout-percent__info">
        <div className="inout-percent__total">Total In: $ 5,000.00</div>
        <div className="inout-percent__done">$450.34</div>
      </div>
      <PercentBar type="out" percent="80%" />
    </div>
  </div>
);

export default InOutPercent;
