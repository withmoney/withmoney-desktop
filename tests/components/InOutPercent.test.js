import renderer from 'react-test-renderer';
import InOutPercent from 'components/InOutPercent';

describe('InOutPercent', () => {
  let Component;

  beforeAll(() => {
    Component = global.setup(InOutPercent);
  });

  it('should render a component correctly', () => {
    const wrapper = renderer.create(Component());

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
