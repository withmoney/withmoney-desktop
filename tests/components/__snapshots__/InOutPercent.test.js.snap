// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InOutPercent should render a component correctly 1`] = `
<div
  className="inout-percent"
>
  <div
    className="inout-percent__type"
  >
    <div
      className="inout-percent__info"
    >
      <div
        className="inout-percent__total"
      >
        Total In: $ 5,000.00
      </div>
      <div
        className="inout-percent__done"
      >
        $450.34
      </div>
    </div>
    <div
      className="percent-bar percent-bar--in"
    >
      <div
        className="percent-bar__progress percent-bar__progress--in"
        style={
          Object {
            "width": "33%",
          }
        }
      />
    </div>
  </div>
  <div
    className="inout-percent__type"
  >
    <div
      className="inout-percent__info"
    >
      <div
        className="inout-percent__total"
      >
        Total In: $ 5,000.00
      </div>
      <div
        className="inout-percent__done"
      >
        $450.34
      </div>
    </div>
    <div
      className="percent-bar percent-bar--out"
    >
      <div
        className="percent-bar__progress percent-bar__progress--out"
        style={
          Object {
            "width": "80%",
          }
        }
      />
    </div>
  </div>
</div>
`;
